transaction/summary/src/verifier.rs

```diff
@@ -165,7 +199,12 @@ impl TxSummaryStreamingVerifierCtx {
             {
                 return Err(Error::AmountVerificationFailed);
             }
-            report.balance_add(TransactionEntity::Swap, token_id.into(), value)?;
+
+            // Add swap output to report
+            // NOTE: this is not exercised as swap rings are created without a transaction
+            // ... does this mean the ocurrence of a swap output is invalid / doesn't need
+            // to be handled here?
```

Comment:
i think its exercised for example in the test you marked TODO, at line 794 above:

```
        // Output to swap counterparty
        (TransactionEntity::Swap, token2, value2),
```

Reply:
ahh i think i have used `output` ambiguously... as part of a swap we will have outputs _to_ the counterparty, but per previous discussion we don't -generate- swaps using the transaction builder so i think there is no case where the output _is_ as swap?